packages/frontend-v2/src/components/molecules/paymentCard.tsx

```diff
@@ -0,0 +1,98 @@
+import { membershipService } from '../../services/membership.service';
+import { paymentService } from '../../services/payment.service';
+import { paymentRequestService } from '../../services/paymentRequest.service';
+
+interface IProps {
+  memberShip: IMembership;
+  onPaymentReceivedFunction: CallableFunction;
+}
+
+export const PaymentCard = (props: IProps) => {
+  const { memberShip, onPaymentReceivedFunction } = props;
+  const [alreadyRequested, setAlreadyRequested] = useState<boolean>(false);
+  useEffect(() => {
+    if (memberShip.paymentRequest) setAlreadyRequested(true);
+  }, [memberShip.paymentRequest]);
+  const createPaymentRequest = async (
+    membership: IMembership
+  ): Promise<void> => {
+    if (!membership.plan) return; // TODO: how to correctly check non nullity of some fields more globally ?
+    try {
+      const newPaymentRequest = await paymentRequestService.add({
+        amount: membership.plan.price,
+        date: new Date(),
+        description: 'demandé manuellement',
+      });
+      await membershipService.update(membership.id, {
+        paymentRequest: newPaymentRequest?.data,
+      });
+      setAlreadyRequested(true);
+    } catch (e) {
+      console.error('error on createPaymentRequest()', e);
+    }
+  };
+  const createPayment = async (memberShip: IMembership): Promise<void> => {
+    if (!memberShip.plan) return;
+    let paymentData: IPayment = {
+      amount: memberShip.plan.price,
+      date: new Date(),
+      hasBeenCanceled: false,
+    };
+
+    try {
+      let newPayment;
+      if (alreadyRequested) {
+        paymentData.paymentRequest = memberShip.paymentRequest;
+        paymentData.member = memberShip.member;
+        newPayment = await paymentService.add(paymentData);
+      } else {
+        newPayment = await paymentService.createPaymentWithoutRequest({
+          payment: paymentData,
+          memberShip,
+        });
+      }
+      if(newPayment) {
+        onPaymentReceivedFunction()
+      }
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+
+  return (
+    <div className="card">
+      <div className="card-header">
+        {memberShip.member?.user?.firstname} {memberShip.member?.user?.lastname}
+      </div>
+      <div className="card-body">
+        <h5 className="card-title">
+          Abonnement {planTypeMapper(memberShip.plan?.type)}, échu le{' '}
+          {memberShip.endDate}
+        </h5>
+        prix : {memberShip.plan?.price}.-
+        <div className="float-right">
+          <button
+            type="button"
+            className="btn btn-secondary"
+            onClick={async () => createPaymentRequest(memberShip)}
+            disabled={alreadyRequested}
+          >
+            Paiment demandé
+          </button>
+          <button
+            type="button"
+            className="btn btn-success"
+            onClick={async () => createPayment(memberShip)}
```

Comment:
you can remove async here as you don't have await into the scope of the anonymous function